CRM/Utils/PagerAToZ.php

```diff
@@ -151,7 +152,17 @@ public static function createLinks(&$query, $sortByCharacter, $isDAO) {
           $element['class'] = "active";
           $klass = 'class="active"';
         }
-        $url = CRM_Utils_System::url($path, "force=1&qfKey=$qfKey&sortByCharacter=");
+        $urlParams = [
+          'force' => 1,
+          'qfKey' => $qfKey,
+        ];
+        if ($query->_context === 'amtg') {
```

Comment:
Oops this is throwing a notice on contribution and event page listings. PR: https://github.com/civicrm/civicrm-core/pull/19515